core/nodepool/config/templates/stack-template.json

```diff
@@ -361,6 +368,7 @@
                   "Action": [
                     "autoscaling:DescribeAutoScalingGroups",
```

Comment:
I still don't understand, what is the benfit of running cluster autoscaler on worker nodes?